openmp/cmake/HandleOpenMPOptions.cmake

```diff
@@ -44,9 +44,9 @@ append_if(OPENMP_HAVE_DATA_SECTIONS "-fdata-sections" CMAKE_C_FLAGS CMAKE_CXX_FL
 
 if (MSVC)
   # Disable "warning C4201: nonstandard extension used: nameless struct/union"
-  append("-wd4201" CMAKE_C_FLAGS CMAKE_CXX_FLAGS)
+  append_if(TRUE "-wd4201" CMAKE_C_FLAGS CMAKE_CXX_FLAGS)
```

Comment:
Using `append_if(TRUE ...)` because on MSVC silencing unknown warnings is a silent action.

Reply:
Can we just use `list(APPEND ...)` here?